inmemory.go

```diff
@@ -0,0 +1,82 @@
+package repohandler
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+// InMemoryDB implements RepoHandler for a custom in memory database
+type InMemoryDB struct {
+	DB *map[string]interface{}
+}
+
+// Create inserts data by creating a generated uuid to track data
+func (in *InMemoryDB) Create(data interface{}) error {
+	uniqueID := uuid.New().String()
+	database := *in.DB
+	database[uniqueID] = data
+
+	return nil
+}
+
+// Get retrieves all data from the map
+func (in *InMemoryDB) Get() ([]interface{}, error) {
+	database := *in.DB
+	var values []interface{}
+
+	for uniqueID := range database {
```

Comment:
while looping make sure to loop from a plural item, database name is a plural form but expecting to have 

`for uniqueID := range uniqueIds {` something like this, which make sense.